### ISSUE TITLE
Does this plugin work in Homebridge 1.1.0, NPM v6.14.5 and Node v12.18.0 

### ISSUE THREAD

**Issue by CyberChris79** [SeydX/homebridge-fritz-platform#85]
Hi,

does this plugin work in Homebridge 1.1.0, NPM v6.14.5 and Node v12.18.0, because i get a warning during the installation, so i uninstalled it again.

Thank you

Christian

**Comment by Laeae** [comment]
same here, the plugin does no longer expose wifi and deflection for me. Have the same warning when loading the plugin.

**Comment by CyberChris79** [comment]
Hi @Laeae ,

the developer told us, to update this plugin in a few weeks.
https://forum.smartapfel.de/forum/thread/5482-homebridge-v1-1-x-support-plugins-von-seydx/

So i'll wait with the installation.

Christian

**Comment by Laeae** [comment]
Thanks Christian. Darn. Then I'll have to evaluate whether there is an option to downgrade the homebridge version.

**Comment by Laeae** [comment]
Does anyone know for sure to which version of homebridge you would have to downgrade, to again gain access to all features?

**Comment by CyberChris79** [comment]
I think you need to downgrade to the Homebridge Version v0.4.53

**Comment by Laeae** [comment]
Thanks, I just tried and it did not bring back my wifi switch nor the deflection switch for some reason. It seems to be not as simple as that. For now I give up and pray that SeydX will have plenty of time to adapt the plugin to the 1.* version of homebridge. 

After all, it's again another lecture in "never change a running system". For some reason I have upgraded homebridge while I knew that something like this could happen. Damn. It's so hard to resist an upgrade.

**Comment by BLoodConstructions** [comment]
Helleluja, what a disaster. Git mine running home bridge version 1.1.0., Node v.12.16.1 und nem v6.14.5. Everything's working great an all switches are visible and working. Greetings

**Comment by CyberChris79** [comment]
Did you also download your Node Version? This is also a must.
I always make an image backup, when i upgrade to a more higher version.

**Comment by BLoodConstructions** [comment]
I'm very careful bout running updates especially on homebridge. Well in my configuration it's running very good. So don't need to upgrade. Maybe in a few months I'll have a look. And yes I still run a backup before updating.

**Comment by CyberChris79** [comment]
Hi @BLoodConstructions ,

i ment @Laeae with my answer ;)

**Comment by Laeae** [comment]
Thanks to you all for trying to help me. 

I've upgraded Node to v12.18.0 and then to its latest version 14, however this did not help unfortunately. 

The tweak with editing the package.node file does not work either. I do not seem to have write access to this file. 

Stuck for now. Anyway, thanks again!

**Comment by Laeae** [comment]
Does anyone know if anything is happening on this topic? Is there any estimated timeline for the necessary changes to be done? I'm very well aware that I'm asking for a free lunch here, however it's just a matter of trying to restart all over (not updating to the latest Homebridge version, or to wait for this plugin to be updated... 

thanks for any hint

**Comment by udovic2100** [comment]
@Laeae: I’m running the plugin with Homebridge Version 1.1.1, Node.js Version 12.18.1 and Npm Version 6.14.5 without any problems so far. A Switch and several thermostats are connected to my FritzBox 7590.

**Comment by Laeae** [comment]
Yeah no, the thermostats do not cause any issues whatsoever. The problem lies with 

- deflection, 
- presence and 
- wifi switches.

I guess you do not use any of these...

**Comment by udovic2100** [comment]
Ok no. I’m only using DECT switches and DECT thermostats.

**Comment by SeydX** [comment]
Im also using this plugin with newest homebridge and node version without these issues u listed.

**Comment by Laeae** [comment]
That leaves me with a bad feeling. You own this plugin and deflection, presence and wifi switches work for you? How can we try to find out what the root cause is for all the others here who do not find this working for them? 

Thanks

**Comment by Laeae** [comment]
I have to say that I have finally resolved the issues, and everything is working fine now. The root cause for me had been a configuration for a repeater. When I had removed this, the other options have worked again fine.

**Comment by CyberChris79** [comment]
Thank you for your feedback.

 

Von: Laeae <notifications@github.com> 
Gesendet: Freitag, 4. September 2020 10:32
An: SeydX/homebridge-fritz-platform <homebridge-fritz-platform@noreply.github.com>
Cc: CyberChris79 <cyberchris79@gmx.de>; Author <author@noreply.github.com>
Betreff: Re: [SeydX/homebridge-fritz-platform] Does this plugin work in Homebridge 1.1.0, NPM v6.14.5 and Node v12.18.0 (#85)

 

I have to say that I have finally resolved the issues, and everything is working fine now. The root cause for me had been a configuration for a repeater. When I had removed this, the other options have worked again fine.
